lib/linguist/languages.yml

```diff
@@ -131,15 +131,13 @@ ASN.1:
   codemirror_mode: asn.1
   codemirror_mime_type: text/x-ttcn-asn
   language_id: 7
-ASP:
+ASP.NET:
   type: programming
-  color: "#6a40fd"
```

Comment:
Given ASP.NET is the language peeps are more likely to use today, we should probably give it a colour instead of falling to the default `#cccccc`

Reply:
@lildude Which color should it be?

Reply:
I have no idea. Normally people aim for a colour related to the language branding.

Reply:
As far as I know, the current branding color for the dotnet platform is `#9400ff`, although it's the same branding for C#, VB.NET, etc.